modules/acpica/Kconfig

```diff
@@ -6,4 +6,10 @@ config ACPI
 
 menu "ACPI driver support"
 
+config ACPI_FULL_SUPPORT
```

Comment:
nah, either we support acpi or we don't. Let's not add intermediate support (which is cryptic as well: what means CONFIG_ACPI alone vs CONFIG_ACPI_FULL_SUPPORT then?)

Reply:
So basically, it's the difference between getting acpica that enables only basic ACPI tables and acpica that enables the same plus DSDT support? 

In that case, name this option ACPI_DSDT_SUPPORT, it's clearer imo.